demo-extended/values.schema.json

```diff
@@ -74,6 +79,124 @@
+              "description": "The number of seconds between each probe"
+            },
+            "failureThreshold": {
+              "title": "Failure Threshold",
+              "type": "integer",
+              "description": "The number of consecutive failures required to consider the probe as failed"
+            }
+          },
+          "order": [
+            "path",
+            "initialDelaySeconds",
+            "periodSeconds",
+            "failureThreshold"
+          ]
+        },
+        "liveness": {
+          "title": "Liveness Probe",
+          "type": "object",
+          "description": "Configuration for the liveness probe",
+          "properties": {
+            "path": {
+              "title": "Path for probe",
+              "type": "string",
+              "description": "The endpoint to probe for liveness"
+            },
+            "initialDelaySeconds": {
+              "title": "Initial Delay Seconds",
+              "type": "integer",
+              "description": "The number of seconds to wait before performing the first probe"
+            },
+            "periodSeconds": {
+              "title": "Period Seconds",
+              "type": "integer",
+              "description": "The number of seconds between each probe"
+            },
+            "failureThreshold": {
+              "title": "Failure Threshold",
+              "type": "integer",
+              "description": "The number of consecutive failures required to consider the probe as failed"
+            }
+          },
+          "order": [
+            "path",
+            "initialDelaySeconds",
+            "periodSeconds",
+            "failureThreshold"
+          ]
+        },
+        "readiness": {
+          "title": "Readiness Probe",
+          "type": "object",
+          "description": "Configuration for the readiness probe",
+          "properties": {
+            "path": {
+              "title": "Path for probe",
+              "type": "string",
+              "description": "The endpoint to probe for readiness"
+            },
+            "initialDelaySeconds": {
+              "title": "Initial Delay Seconds",
+              "type": "integer",
+              "description": "The number of seconds to wait before performing the first probe"
+            },
+            "periodSeconds": {
+              "title": "Period Seconds",
+              "type": "integer",
+              "description": "The number of seconds between each probe"
+            },
+            "failureThreshold": {
+              "title": "Failure Threshold",
+              "type": "integer",
+              "description": "The number of consecutive failures required to consider the probe as failed"
+            }
+          },
+          "order": [
+            "path",
+            "initialDelaySeconds",
+            "periodSeconds",
+            "failureThreshold"
+          ]
+        }
+      },
+      "order": [
+        "enabled",
```

Comment:
can we add `enabled` to each probe separately? Also can you make it disabled by default in `values.yaml`

Reply:
Hi @petar-cvit , actually initially I tried to have separate toggle for each probe. But it was not working fine. Maybe will try it again.

Reply:
Please do. You can paste the error here if you want and we can try to figure it out

Reply:
Can you explain me this error. 
![image](https://github.com/cyclops-ui/templates/assets/110477025/1df8867a-c40e-4ad5-821f-074d8dd5240c)

UI changes performed : 
![image](https://github.com/cyclops-ui/templates/assets/110477025/002a00f9-9b31-4a41-8af4-98a8ce1fb6c7)

Reply:
We refactored manifest rendering so you can try out the template again with the updated version.